src/app/src/main/resources/config.properties

```diff
@@ -5,7 +5,7 @@ adr_pop_schema=inputs-unaids-population
 adr_shape_schema=inputs-unaids-geographic
 adr_survey_schema=inputs-unaids-survey
 adr_schema=inputs-unaids-estimates
-adr_url=https://dev.adr.fjelltopp.org/
+adr_url=https://adr.unaids.org/me/
```

Comment:
We also don't want to hardcode `/me` at the end of the url here because this url is used in various places, not just for the link to get the access code.  The link to the access code should just be updated in the `ADRKey.vue` component.